R/derive_vars_aage.R

```diff
@@ -93,6 +93,9 @@ derive_vars_aage <- function(dataset,
   start_date <- assert_symbol(enexpr(start_date))
   end_date <- assert_symbol(enexpr(end_date))
   assert_data_frame(dataset, required_vars = expr_c(start_date, end_date))
+
+  age_unit <- valid_time_mappings(age_unit)
```

Comment:
This could create confusing error messages. For example
```
> derive_vars_aage(data, age_unit = "wks")
# A tibble: 1 × 4
  BRTHDT     RANDDT      AAGE AAGEU
  <date>     <date>     <dbl> <chr>
1 1984-09-06 2020-02-24  1850 weeks
```
works. But with a typo:
```
> derive_vars_aage(data, age_unit = "wls")
Error in `assert_character_scalar()` at admiral/R/derive_vars_aage.R:99:2:
! `age_unit` must be one of 'years', 'months', 'weeks', 'days', 'hours', 'minutes' or 'seconds' but is 'wls'
```
the error message suggests that `"wks"` is not valid.

Furthermore, the unit variable should be set to the specified value. I.e., in the first example `AAGEU` should be `"wks"` instead of `"weeks"`.